skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +335,92 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
+    def test_ancom_basic(self):
```

Comment:
Can you add a test for passing a grouping vector with more than 2 groups (assuming you're using a test function that works with 2+ groups)?